nirc_ehr/resources/queries/dbo/q_flags.sql

```diff
@@ -0,0 +1,45 @@
+-- Quarantine
+SELECT anmEvt.ANIMAL_EVENT_ID                                                    AS objectid,
+       anmEvt.ANIMAL_ID.ANIMAL_ID_NUMBER                                         AS Id,
+       CAST(anmEvt.EVENT_DATETIME AS TIMESTAMP)                                  AS startdate,
+       CAST(ae.EVENT_DATETIME AS TIMESTAMP)                                      AS enddate,
+       (CASE
+            WHEN anmEvt.STAFF_ID.EMAIL_ADDRESS IS NULL THEN 'unknown'
+            ELSE substring(anmEvt.STAFF_ID.EMAIL_ADDRESS, 1,
+                           locate('@', anmEvt.STAFF_ID.EMAIL_ADDRESS) - 1) END)  AS performedby,
+       anmEvt.EVENT_ID                                                           AS flag,
+       CASE WHEN anmCmt.TEXT IS NOT NULL THEN (anmCmt.TEXT || '-' || ac.TEXT)
+       ELSE ac.TEXT END                                                          AS remark,
+       CAST(COALESCE(adt.modified, anmEvt.CREATED_DATETIME) AS TIMESTAMP) AS modified
+FROM ANIMAL_EVENT anmEvt
+         LEFT JOIN ANIMAL_EVENT ae ON ae.ANIMAL_ID = anmEvt.ANIMAL_ID AND ae.EVENT_ID.EVENT_ID = 1674
+         LEFT JOIN ANIMAL_EVENT_COMMENT anmCmt ON anmEvt.ANIMAL_EVENT_ID = anmCmt.ANIMAL_EVENT_ID
+         LEFT JOIN ANIMAL_EVENT_COMMENT ac ON ae.ANIMAL_EVENT_ID = ac.ANIMAL_EVENT_ID
+         LEFT JOIN q_modified_event adt ON anmEvt.ANIMAL_EVENT_ID = adt.event_id
+
+WHERE anmEvt.EVENT_ID.EVENT_ID = 1673
```

Comment:
A comment about 1674 being the ending event and 1673 being the beginning event would be nice